requirements.txt

```diff
@@ -1,4 +1,8 @@
-ops == 2.5.0 # Revert when https://github.com/canonical/operator/issues/984 is fixed.
-ops-lib-pgsql >= 1.4
-requests == 2.28.1
-responses == 0.21.0
+cosl
```

Comment:
I think these requirements have changed incorrectly. Currently in `livepatch-server` the list looks like
```
ops == 2.5.0 # Revert when https://github.com/canonical/operator/issues/984 is fixed.
ops-lib-pgsql >= 1.4
requests == 2.28.1
responses == 0.21.0
```
I don't believe we need pytest and pytest_asyncio and juju. I'm curious why cosl was added too.

Reply:
I'm okay with the latest requests and responses, and we can stick with the latest ops and ops-lib-pgsql too then. But we should remove the other unneeded requirements like `juju`, `pytest`, etc.

Reply:
updated as you requested @kian99